content/includes/try-it-out-custom.html

```diff
@@ -0,0 +1,117 @@
+<div class="ui container">
+    <div class="ui horizontal divider">Try it out</div>
+</div>
+
+<section id="try-it-out">
+    <div class="ui padded container">
+      <h2 class="ui huge center aligned header">
+        Upgrade your documentation with Read the Docs
+      </h2>
+
+      <div class="ui very padded centered grid">
+
+        <div class="ten wide computer sixteen wide tablet sixteen wide mobile column">
+          <div class="ui large left aligned basic vertical segment">
+
+            {# Stylized code blocks #}
+            {% call technical.code_block(title="<code>.readthedocs.yaml</code>", icon="fa-file") %}
+            {% markdown %}
+
+            ```yaml
+            # Read the Docs configuration file for Sphinx projects
+            # See https://docs.readthedocs.io/en/stable/config-file/v2.html for details
```

Comment:
We should follow the same pattern we already have in our "Add project" wizard form with a `tabmenu` and using the same examples: https://github.com/readthedocs/ext-theme/blob/0360c0e3a5a1219d97a3df035163482f5626d080/readthedocsext/theme/templates/projects/import_config.html#L26-L37

Reply:
It seems if we're just putting the configs for all the tools here, it feels a bit weird? I think we just want to highlight we can run any tool, and not copy all the configs again here?

Reply:
Ah, OK. Then, we should stick with one working example here only instead of using Python and Node in the same config outputting HTML files to the same directory.

Reply:
I was thinking "more in general" where we are probably merging all the "Supported doctools" in only one page with all the examples together and a selector for the doctool. That way we don't need a specific page per doctool, and we can link to the page in our docs for each specific doctool -- that will explain things in deep.